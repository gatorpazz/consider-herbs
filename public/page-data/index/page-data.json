{"componentChunkName":"component---src-pages-index-js","path":"/","result":{"data":{"contentfulSiteSettings":{"heroTitle":"Get back to your nature.","heroBio":"Consider the holistic power of herbs and experience the healthy effects of our earth.","callToActionButton":"Explore Today"}},"pageContext":{}},"staticQueryHashes":[]}