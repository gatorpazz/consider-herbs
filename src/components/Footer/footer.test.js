import React from "react"
import renderer from "react-test-renderer"
import { PureFooter as Footer } from "./index"

describe("Footer", () => {
  it("renders correctly", () => {
    const data = {
      contentfulSiteSettings: {
        disclaimer: {
          internal: {
            content: "The information presented herein by Consider Herbs is intended for educational purposes only. These statements have not been evaluated by the FDA and are not intended to diagnose, cure, treat or prevent disease. Individual results may vary, and before using any supplements, it is always advisable to consult with your own healthcare provider."
          }
        }
      }
    }
    const tree = renderer
      .create(<Footer data={data} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})