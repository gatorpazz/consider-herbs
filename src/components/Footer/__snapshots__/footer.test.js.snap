// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="bg-gray-100 object-bottom"
>
  <div
    className="container mx-auto px-10 pt-10 pb-6"
  >
    <div
      className="flex flex-wrap"
    >
      <div
        class="w-full md:w-1/4 text-left"
      >
        <h5
          className="mb-4 font-bold"
        >
          Links
        </h5>
        <ul
          className="mb-4 font-semibold"
        >
          <li
            className="mt-2"
          >
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li
            className="mt-2"
          >
            <a
              href="/shop/"
            >
              Shop
            </a>
          </li>
          <li
            className="mt-2"
          >
            <a
              href="/blog/"
            >
              Blog
            </a>
          </li>
          <li
            className="mt-2"
          >
            <a
              href="/about/"
            >
              About
            </a>
          </li>
          <li
            className="mt-2"
          >
            <a
              href="/contact/"
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div>
      <p
        className="mt-4 text-left text-xs"
      >
        The information presented herein by Consider Herbs is intended for educational purposes only. These statements have not been evaluated by the FDA and are not intended to diagnose, cure, treat or prevent disease. Individual results may vary, and before using any supplements, it is always advisable to consult with your own healthcare provider.
      </p>
    </div>
    <div>
      <h3
        className="mt-4 text-left"
      >
        Consider Herbs Â© 2021
      </h3>
    </div>
  </div>
</footer>
`;
